packages/storybook/src/nimble/table/table.mdx

```diff
@@ -97,6 +97,71 @@ The expected usage of the dynamically loaded hierarchy is as follows:
 -   calling `setData` will clear options associated with IDs that are no longer
     present in the data
 
-{/* ## Accessibility */}
+## Accessibility
+
+### Keyboard Navigation
```

Comment:
I've started standardizing on sentence casing for mdx headings
```suggestion
### Keyboard navigation
```

Reply:
Updated